### PR TITLE
Add static organisation sub-header for Google

### PR DESCRIPTION
**Pull request by i-like-robots**
For Google:

screen shot 2015-05-20 at 17 46 03

For not Google:

screen shot 2015-05-20 at 17 45 44

### CODE REVIEW

**Comment by matthew-andrews** [comment]
nice…… but i think i caused some conflicts for you …… sorry :disappointed:

File: views/partials/sub-header-organisation.html [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+			1998, Mountain View, CA, USA
+		</dd>
+		<dt class="sub-header__summary__title">
+			<abbr title="Subsidiaries">Sub</abbr>
+		</dt>
+		<dd class="sub-header__summary__detail">
+
+			<ul class="sub-header__link-list">
+				<li class="sub-header__link-list__item">
+					<a href="" data-trackable="subsidiary">DoubleClick, Inc.</a>
```
Comment (i-like-robots) [inline views/partials/sub-header-organisation.html]: Haven't found links that work yet for these companies on Next

File: views/partials/sub-header-organisation.html [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+<aside class="sub-header__main__section" data-o-grid-colspan="12 M4 L5">
+	<h3 class="sub-header__main__heading">Summary</h3>
+
+	<dl class="sub-header__summary" data-trackable="organisation-summary">
+		<dt class="sub-header__summary__title">
+			<abbr title="Chief Executive Officer">CEO</abbr>
+		</dt>
+		<dd class="sub-header__summary__detail">
+			Lawrence E. Page
```
Comment (commuterjoy) [inline views/partials/sub-header-organisation.html]: hyperlink?

**Comment by matthew-andrews** [comment]
in #450

**Pull request closed without merging**